src/site/content/en/animations/animations-examples/index.md

```diff
@@ -0,0 +1,346 @@
+---
+layout: post
+title: Examples of high-performance CSS animations
+authors:
+  - rachelandrew
+description: |
+  Demonstrating how high performance techniques can create complex and beautiful animations.
+date: 2020-10-20
+tags:
+  - animations
+  - performance
+---
+
+In this post find out how some popular animations found on CodePen have been created.
+THese animations all use the performant techniques discussed in other articles in this section.
+
+See [Why are some animations slow?](/animations-overview/) to learn the theory behind
+these recommendations, and the [Animations Guide](/animations-guide) for practical tips.
+
+## Wizard loading animation
+
+{% IFrame {
+  src: 'https://codepen.io/iamlark/embed/jYzYJg?height=458&theme-id=light&default-tab=result',
+  title: 'Only CSS Loader - Wizard by Guilmain Dorian'
+}
+%}
```

Comment:
```suggestion
<div style="height: 500px; width: 100%;">
  {% IFrame {
    src: 'https://codepen.io/iamlark/embed/jYzYJg?height=458&theme-id=light&default-tab=result',
    title: 'Only CSS Loader - Wizard by Guilmain Dorian'
  } %}
</div>
```

Looks like we need to explicitly set a height:

![image](https://user-images.githubusercontent.com/4713486/96823098-86cb3b00-13e0-11eb-9846-a531748e155d.png)

The linter is angry with these iframe components, maybe wrapping them in divs or reformatting will help…

![image](https://user-images.githubusercontent.com/4713486/96823223-ddd11000-13e0-11eb-966f-8389677eae05.png)